Index.html

```diff
@@ -287,6 +287,47 @@ <h2 class="about-title">About me</h2>
     </ul>
 
   </section>
+
+  <section class="contact-form">
+
+    <p class="contact-form-description">I'm always <br>
+      interested in hearing <br> 
+      about new projects, <br>
+      so if you'd like to <br> 
+      chat please get in touch.
+    </p>
+
+    <form class="form" action="https://formspree.io/f/mzbooyno" method="post">
+      <label class="first-name-label" for="fname">Full name</label>
+      <input class="form-input" maxlength="30" type="text" name="Full Name" required>
+      <label class="email-label" for="email">Email address</label >
+      <input class="form-input" type="email" name="Email" required>
```

Comment:
- [ ] The Figma template form inputs look like there are no labels over them. The 'Full name' and 'Email address' placeholders should disappear when the user fills in those inputs. They should be only placeholder texts. Your style matches the template perfectly, kindly try to remove the labels to placeholder texts without changing the CSS style you created.
 -
<img width="395" alt="image" src="https://user-images.githubusercontent.com/61919617/185979133-5e668272-12c7-4785-b05e-ef88a27c7e81.png">
<img width="395" alt="image" src="https://user-images.githubusercontent.com/61919617/185979132-b615adad-043a-4ae1-adfe-5df9e47136e4.png">